client/src/serviceWorker.js

```diff
@@ -0,0 +1,137 @@
+// This optional code is used to register a service worker.
```

Comment:
do we need service workers right now?

Reply:
No, it is generated by standard create-react-app. Will remove.